packages/web-runtime/src/components/SidebarNav/SidebarNavItem.vue

```diff
@@ -10,8 +10,11 @@
       :data-nav-id="index"
       :data-nav-name="navName"
     >
-      <oc-icon :name="icon" :fill-type="fillType" />
-      <span class="oc-ml-m text" :class="{ 'text-invisible': collapsed }" v-text="name" />
+      <span class="oc-flex">
+        <oc-icon :name="icon" :fill-type="fillType" />
+        <span class="oc-ml-m text" :class="{ 'text-invisible': collapsed }" v-text="name" />
+      </span>
+      <oc-tag class="oc-pt-rm oc-pb-rm" v-if="tag">{{ tag }}</oc-tag>
```

Comment:
```suggestion
      <oc-tag v-if="tag" class="oc-py-rm">{{ tag }}</oc-tag>
```
Two nitpicks:
- 1: I personally like to have the v-if/else etc directives before all the attributes and listeners. When debugging code that allows you to mentally skip as early as possible.